.github/workflows/push.yaml

```diff
@@ -235,3 +235,27 @@ jobs:
           architecture: x64
       - name: Building for ios
         run: flutter build ios --release --no-codesign
+        # '--no-codesign' is used for building without code signing.
```

Comment:
Rename:

1. Workflow to 'ios-build-and-release' (with appropriate casing of course)
2. Built ipa to 'app-release.ipa' to ensure consistency with the apk.

Reply:
@literalEval made changes sir.

Reply:
1. I meant the whole iOS Build workflow; not the specific one.